### PR TITLE
Add a test for CsvTransactionsExporter

### PR DESCRIPTION
**Pull request by hades**
Also a minor change to OfxExporterTest.

### CODE REVIEW

File: app/src/test/java/org/gnucash/android/export/csv/CsvTransactionsExporterTest.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package org.gnucash.android.export.csv
```
Comment (pnemonic78) [inline app/src/test/java/org/gnucash/android/export/csv/CsvTransactionsExporterTest.kt:1]: package org.gnucash.android.test.unit.export

File: app/src/test/java/org/gnucash/android/test/unit/export/OfxExporterTest.java [hunk 2]
```diff
@@ -53,7 +53,7 @@
 @RunWith(RobolectricTestRunner.class)
 //package is required so that resources can be found in dev mode
 @Config(sdk = 21, shadows = {ShadowCrashlytics.class, ShadowUserVoice.class})
-public class OfxExporterTest extends BookHelperTest {
+public class OfxExporterTest {
```
Comment (pnemonic78) [inline app/src/test/java/org/gnucash/android/test/unit/export/OfxExporterTest.java:56]: why?

**Review by pnemonic78 — CHANGES_REQUESTED** [review]

File: app/src/test/java/org/gnucash/android/export/csv/CsvTransactionsExporterTest.kt (hunk 1 above)
Comment (hades) [reply app/src/test/java/org/gnucash/android/export/csv/CsvTransactionsExporterTest.kt]: Done.

File: app/src/test/java/org/gnucash/android/test/unit/export/OfxExporterTest.java (hunk 2 above)
Comment (hades) [reply app/src/test/java/org/gnucash/android/test/unit/export/OfxExporterTest.java:56]: It's not using anything from BookHelperTest.

**Review by pnemonic78 — APPROVED** [review]

**Pull request merged**